index.js

```diff
@@ -0,0 +1,36 @@
+/* global module */
+/* eslint-disable no-var */
+
+'use strict'
+
+var modules = [
+  'model/Builtin',
+  'model/Codec',
+  'model/Field',
+  'model/Model',
+  'AsyncStream',
+  'Client',
+  'errors',
+  'objects',
+  'PageStream'
+  // query handled at bottom
+]
+
+module.exports = Object.assign.apply(null, modules.map(function(name) {
+  var module = require('./lib/' + name)
+  if (typeof module === 'object') {
+    var dfault = module.default
+    if (dfault !== undefined) {
+      delete module.default
+      module[dfault.name] = dfault
+    }
+    return module
+  } else {
+    var obj = {}
+    obj[module.name] = module
+    return obj
+  }
+}))
```

Comment:
I'd prefer that we just set the exports manually rather than through this helper. There's not very many, and its much clearer to do it by hand than to try to infer the shape of the underlying module.